tests/Unit/LocalFileVolumeContentSizeTest.php

```
@@ -64,3 +64,62 @@
     expect($array)->toHaveKey('is_too_large');
     expect($array['is_too_large'])->toBeTrue();
 });
+
+it('does not read regular bind-mounted file contents while loading service settings', function () {
+    $helpers = file_get_contents(base_path('bootstrap/helpers/services.php'));
+    $filesystemSync = str($helpers)
```

PHPStan (2.2.7)

[warning] Parameter `#1` $string of function str expects string|null, string|false given.


(argument.type)